server.js

```diff
@@ -1,7 +1,39 @@
-const express = require('express')
-const app = express()
-const PORT = 5000
+const express = require('express');
+const mongoose = require('mongoose');
+require('dotenv').config();
+const app = express();
+const PORT = 5000;
+const cors = require('cors');
 
+//Middleware
+app.use(express.json());
+app.use(cors());
+
+//Start server
 app.listen(PORT, () => {
-    console.log(`Backend-Server started successfully at port: ${PORT}`)
-})
+  console.log(`Backend-Server started successfully at port: ${PORT}`);
+});
```

Comment:
I think if we are using `.env` variables we could put our PORT there too, that way is not so visible and it is not something that is in the file somewhere and one knows exactly where to find. What do you think?

Reply:
good idea. included.